src/media/media.service.spec.ts

```diff
@@ -113,7 +122,7 @@ describe('MediaService', () => {
     });
 
     it('returns the image id and filehandle', async () => {
-      const imageMock = { id: 'mocked-id', identifier: 'mocked-identifier' };
+      const imageMock = imageMocks[0];
       jest.spyOn(imageRepositoryMock, 'save').mockReturnValue(imageMock);
 
       const result = await mediaService.uploadImage(userMock, fileMock);
```

Comment:
Not sure if the compare view is just scrambled up but there should be some kind of test here? Like expect(result).toEqual(something)?

Reply:
Yes, seems to be scrambled, the assertion is below :)